dolphinscheduler-python/pydolphinscheduler/DEVELOP.md

```diff
@@ -83,6 +83,23 @@ black .
 flake8
 ```
 
+## Build Docs
+We use [sphinx][sphinx] to build docs. DS Python API CI would automatically build docs when you submit pull request in 
+GitHub. You may locally ensure docs could be built suceessfully in case the failure blocks CI.
+
+To build docs locally, install sphinx and related python modules first via:
+```shell
```

Comment:
Same as the title, we should add a blank line between content and code

Reply:
Same as L98

Reply:
Sure, will fix it.